packages/driver/src/cypress/stack_utils.ts

```diff
@@ -60,6 +60,58 @@ const stackWithLinesRemoved = (stack, cb) => {
   return unsplitStack(messageLines, remainingStackLines)
 }
 
+const stackWithWrappingLinesRemoved = (stack, specWindow) => {
+  const modifiedStack = stackWithLinesRemoved(stack, (lines) => {
+    // Guard against Cypress being undefined/null (can happen when users quickly reload tests)
+    if (!specWindow?.Cypress) {
+      return lines
+    }
+
+    if (specWindow.Cypress.isBrowser('chrome')) {
```

Comment:
This needs to be any Chromium browser:

```suggestion
    if (specWindow.Cypress.isBrowser({ family: 'chromium' })) {
```